settings/settings.json

```
@@ -335,6 +335,13 @@
                                         "id": "MoviesGrid"
                                     }
                                 ]
+                            },
+                            {
+                                "title": "Disable Community and Critical Ratings",
+                                "description": "Hide the star, community rating, tomato, and critical rating for Movies.",
```

```suggestion
                                "description": "Things that describe how good the Movie is i.e. star, community rating, tomato, and critical rating",
```


There's probably a better way to word this

---

Went with "Show ratings for how good a movie is." Open to suggestions.

---

In my mind the setting description describes the setting title so IMO  `Ratings for how good a movie is.` would be a little better but that might nitpicking.